src/core/viz/expressions/torque.js

```diff
@@ -163,55 +186,58 @@ export class Torque extends BaseExpression {
      * @instance
      * @name getSimTime
      */
+    
     getSimTime() {
-        const c = this._cycle.eval(); //from 0 to 1
-
+        const progress = this.progress.eval(); //from 0 to 1
         const min = this.input.min.eval();
         const max = this.input.max.eval();
 
-        if (!(this.input.min.eval() instanceof Date)) {
-            return min + c * (max - min);
+        if (!(min instanceof Date)) {
+            return progress * (max - min) + min;
         }
 
-
         const tmin = min.getTime();
```

Comment:
@davidmanzanares @elenatorro is the .getTime() call ok? I think max is already a number